servicebox-app/src/main/java/io/halkyon/service/ClaimResource.java

```diff
@@ -0,0 +1,15 @@
+package io.halkyon.service;
+
+import io.halkyon.model.Claim;
+import io.quarkus.hibernate.orm.rest.data.panache.PanacheEntityResource;
+import io.quarkus.rest.data.panache.ResourceProperties;
+
+
+/***
+ * The "@ResourceProperties" annotation of the Quarkus JAX-RS panache extension generates the JAX-RS CRUD endpoints for the ClaimResource interface.
```

Comment:
`@ResourceProperties` annotation that can be used to customize certain features of the resource (here the path) but is not responsible for generating the Resource implementation.
REST Data with Panache generates JAX-RS resources based on the presence of the interface which extends the PanacheEntityResource<Claim, Long>, being `Claim`the entity.

Reply:
do you then suggest to review the wording of the javadoc ?

Reply:
```suggestion
/***
 * REST Data with Panache extension generates JAX-RS resources based on the presence of this interface.
 * The @ResourceProperties annotation is used to customize the path of the resource.
 * More information is available within the quarkus documentation: <a href="https://quarkus.io/guides/rest-data-panache">https://quarkus.io/guides/rest-data-panache</a>
 * The endpoints generated support as HTTP request/response, the `application/json` content type.
 * The endpoints can be overridden, extended using the @see io.halkyon.template.ClaimPage class.
 */
```